examples/benzeneselenol.cjson

```diff
@@ -0,0 +1,60 @@
+{
+  "chemical json": 0,
+  "name": "Benzeneselenol",
+  "inchi": "1S/C6H6Se/c7-6-4-2-1-3-5-6/h1-5,7H",
+  "formula": "C 6 H 6 Se",
+  "atoms": {
+    "elements": {
+      "number": [  6,  6,  6,  6,  6,  6,  1,  1,  1,  1,  1, 34,  1 ]
+    },
+    "coords": {
+      "3d": [  -2.894762	-0.013642	 -0.001528
+               -2.206251	 1.193851	 -0.002521
+               -0.816426	 1.215302	 -0.002201
+               -0.103352	 0.018392	  0.003106
+               -0.790663	-1.194384	  0.00585
+               -2.179957	-1.205971	  0.001789
+               -3.975843	-0.025301  -0.002904
+               -2.750234	 2.129137	 -0.005276
+               -0.296184	 2.163018	 -0.007326
+               -0.247467	-2.130231	  0.013226
+               -2.702896	-2.153075	  0.003664
+                1.821080  -0.043378	 -0.003876
+                2.004358	 1.410007	  0.103449 ]
+    },
+    "atomcharges": {
+      "mulliken": [ -0.49915,  0.056965,  0.172161,  0.349794, -0.153072,  0.094583,
+                     0.016487,  0.050249,  0.002149,  0.01161 ,  0.053777, -0.173671,
+                     0.018118]
+    }
+  },
+  "bonds": {
+    "vibfreqs": [  129.5512   170.6681   231.4278   304.8614   407.8299   472.5026
+                   629.9087   679.9032   693.2509   746.7694   812.5113   850.2578
+                   915.8742   987.1252   988.1785  1002.8922  1038.1073  1091.4005
+                   1102.3417  1183.3857  1209.2727  1311.3497  1355.6441  1471.4447
+                   1510.1919  1611.9088  1619.0156  2391.2487  3165.1596  3171.3909
+                   3182.0753  3188.5786  3198.0359 ]
+    "vibsyms": [  u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
```

Comment:
'A' is the symmetry of the vibration. The molecule has low or no point group symmetry, I.e. the only irrep is 'A'. 

Vibrations are in an odd spot.

> On Apr 11, 2016, at 7:07 AM, Marcus D. Hanwell <notifications@github.com> wrote:
> 
> In examples/benzeneselenol.cjson:
> 
> > +                2.004358	 1.410007	  0.103449 ]
> > +    },
> > +    "atomcharges": {
> > +      "mulliken": [ -0.49915,  0.056965,  0.172161,  0.349794, -0.153072,  0.094583,
> > +                     0.016487,  0.050249,  0.002149,  0.01161 ,  0.053777, -0.173671,
> > +                     0.018118]
> > +    }
> > +  },
> > +  "bonds": {
> > +    "vibfreqs": [  129.5512   170.6681   231.4278   304.8614   407.8299   472.5026
> > +                   629.9087   679.9032   693.2509   746.7694   812.5113   850.2578
> > +                   915.8742   987.1252   988.1785  1002.8922  1038.1073  1091.4005
> > +                   1102.3417  1183.3857  1209.2727  1311.3497  1355.6441  1471.4447
> > +                   1510.1919  1611.9088  1619.0156  2391.2487  3165.1596  3171.3909
> > +                   3182.0753  3188.5786  3198.0359 ]
> > +    "vibsyms": [  u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
> Can't these be standard strings in the array? What does 'A' mean?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>